### PR TITLE
[RB] Fix a couple bazel button bugs

### PR DESCRIPTION
**Pull request by maggie-lou**
* Add a missing space in a command
* Only add bazel flags to bazel commands
* Support platform properties
* For the bb explain button, use commit shas rather than branches when comparing 2 invocations
**Related issues**: N/A

### CODE REVIEW

File: app/util/remote_runner.tsx [hunk 1]
```diff
@@ -53,6 +69,16 @@ export function triggerRemoteRun(invocationModel: InvocationModel, command: stri
     });
 }
 
+// commandWithRemoteRunnerFlags adds useful flags to bazel commands run on a remote runner
+export function commandWithRemoteRunnerFlags(command: string): string {
+  // The buildbuddy_ configs point to the corresponding env that created the workflow
+  // action (Ex. If it was created in dev, points to the dev app)
+  const addlFlags =
```
Comment (jdhollen) [inline app/util/remote_runner.tsx:76]: can we document the connection between this and `ci_runner/main.go` both here and there? this is kind of a brittle connection right now.

**Review by jdhollen — APPROVED** [review]

File: app/util/remote_runner.tsx (hunk 1 above)
Comment (jdhollen) [reply app/util/remote_runner.tsx:76]: (fine to do separate from this pr)

File: app/util/remote_runner.tsx [hunk 2]
```diff
@@ -53,6 +69,16 @@ export function triggerRemoteRun(invocationModel: InvocationModel, command: stri
...
+    "--remote_cache_compression --config=buildbuddy_bes_backend --config=buildbuddy_bes_results_url --config=buildbuddy_remote_cache";
```
Comment (bduffany) [inline app/util/remote_runner.tsx:77]: hm I thought we applied the BES flags by default? maybe only the remote cache ones are needed? otherwise lgtm

**Review by bduffany — APPROVED** [review]

File: app/util/remote_runner.tsx (hunk 2 above)
Comment (maggie-lou) [reply app/util/remote_runner.tsx:77]: The problem is we have this in our .bazelrc

common --bes_results_url=https://app.buildbuddy.io/invocation/
common --bes_backend=grpcs://remote.buildbuddy.io

File: app/util/remote_runner.tsx (hunk 1 above)
Comment (maggie-lou) [reply app/util/remote_runner.tsx:77]: Good idea - done

**Pull request merged**